file_object.rb

```diff
@@ -0,0 +1,30 @@
+require 'json'
+
+class FileObject
+  attr_reader :file_name, :deletion_date, :initial_path
+
+  def initialize(file_name, deletion_date, initial_path)
+    @file_name = file_name
+    @deletion_date = deletion_date
+    @initial_path = initial_path
+  end
+
+  def to_json
+    {
+      file_name: @file_name,
```

Comment:
as you already have `attr_reader`, you can use your instance variables without `@`. So here it can be:
```ruby
file_name: file_name,
```
and etc